src/client/app/components/dashboard/dashboard.component.spec.ts

```diff
@@ -62,7 +62,7 @@ describe('Component: Dashboard', () => {
                     fixture.detectChanges();
 
                     let heroes = fixture.componentInstance.heroes;
-                    expect(heroes.length).toEqual(5);
+                    expect(heroes.length).toEqual(4);
                 });
             })));
     });
```

Comment:
I see we're still using fixture based tests. Did you try the simple way after fixing the mocks and mocking router?